components/search/MoreFacetsDropdown.vue

```diff
@@ -45,7 +53,28 @@
         type: Array,
         default: () => []
       }
+    },
+
+    data() {
+      return {
+        selected: {}
+      };
+    },
+
+    computed: {
+      selectedAmount() {
+        let sum = 0;
+        for (const key of Object.keys(this.selected)) {
```

Comment:
`        for (const key in this.selected) {`

Reply:
`this.selected` is not iterable

Reply:
if you use 'in' instead of 'of' it'll work.